### DISCUSSION TITLE
fltk: FLUID binaries

### DISCUSSION THREAD

**Discussion by splisp** [microsoft/vcpkg#34712] [General]
Hi, I have installed the fltk package in vcpgk and it works correctly. I'd like to use the FLUID tool (fltk's designer) but no binaries are provided; it should be built when building the library from sources, as stated in fltk's docs. So I searched it in vcpkg's build tree and found it. Actually there's one build for each configuration, I think I'd use the one in `vcpkg\buildtrees\fltk\x64-windows-static-rel\bin\FLUID.exe`.

Is there some better practice than using the exe just from the build tree, or to copy it to some Programs folder?

Fabrizio

**Comment by StarGate-One** [comment]
vcpkg should have put it in the  `vcpkg\installed\x64-windows-static-rel\tools\fltk\FLUID.exe` directory/folder.

**Reply by splisp** [reply]
Great, thanks, I hadn't noticed the `tools` folder (I'm quite new to vcpkg)
